lib/net.js

```diff
@@ -1174,11 +1176,27 @@ function Server(options, connectionListener) {
     configurable: true, enumerable: false
   });
 
+  Object.defineProperty(this, '_handle', {
+    set: function(val) {
+      if (val === null) {
+        this[serverListenHasBeenCalled] = false;
+      }
+      this[serverHandle] = val;
+    },
+    get: function() {
+      return this[serverHandle];
+    },
+    enumerable: true
+  });
```

Comment:
I'd stick with simple properties.  Accessors have overhead.  If that means duplicating an assignment in two places, that's something I can live with.